styles.css

```diff
@@ -1,215 +1,39 @@
-@import url(./styles/main.css);
-
-
-/* ***** page layout *** */
-
-.page-layout{
+.hero{
+    padding: 4rem;
     display: grid;
-    grid-template-columns: 1fr 3fr 0.3fr;
+    grid-template-columns: 10rem 1fr 1fr 1rem;
     grid-template-areas: 
-    "header  header header"
-    "sidebar  main   .";
-}
-
-
-/* ***** header ****** */
-
-.header{
-    grid-area: header;
-    padding: 1rem 2.8rem;
-    display: flex;
-    position: sticky;
-    top: 0;
-    z-index: 3;
-    align-items: center;
-    justify-content: space-between;
-    box-shadow: 0 10px 8px -8px #d6d6d6;
-    font-weight: 500;
-    background-color: var(--primary-color)
-}
-
-.header__logo{
-    display: flex;
-    margin-left: 7rem;
-}
-.header__logo > img{
-    width: 15rem;
-    height: 5.8rem
-}
-
-.header__links{
-    list-style: none;
-    display: flex;
-    font-size: 2.9rem;
-    align-items: center;
-    margin-right: 4rem;
-}
-
-.header__links li i{
-    color: black;
+    ". left right .";
 }
 
-.header__links li i:hover{
-    cursor: pointer;
-    /* color: var(--primary-dark-color) */
-    color: var(--secondary-color)
-}
-
-
-/* ****** Sidebar ******* */
-
-.sidebar-main{
-    position: relative;
-    grid-area: sidebar;
-    
-}
+.hero > div {
+    padding: 1rem;
 
-.sidebar{
-    padding: 3rem;
-    padding-left: 4.5rem;
-    padding-right: 3rem;
-    position: sticky;
-    overflow-y: auto;
-    top: 7.5rem;
-    height: calc(100vh - 8.125rem);
-    display: flex;
-    flex-direction: column;
-    gap: 1rem;
-    
 }
 
-.sidebar::-webkit-scrollbar {
-    display: none;
-  }
-
-.sidebar__links{
+.hero-left{
+    grid-area: left;
     display: flex;
     flex-direction: column;
-    align-items: center;
-    gap: 1rem;
-    position: relative;
-}
-
-.sidebar__links::after{
-    content: "";
-    position: absolute;
-    border-right: 2.5px solid var(--primary-color);
-    top: -14rem;
-    right: -0.9rem;
-    height: 145%;
-}
-
-.sidebar h3{
-    display: grid;
     justify-content: center;
-    
+    align-items: flex-start;
+    width: 40vw;
 }
 
-.sidebar a{
-    width: 100%;
-    font-weight: 500;
+.hero-left > a {
+    width: 18rem;
+    height: 4.5rem;
     font-size: 1.8rem;
-    color: var(--dark-color);
-    margin-top: 0.5rem;
-    margin-bottom: 0.5rem;
-    padding: 0.5rem;
-    display: grid;
-    justify-content: center;
+    text-align: center;
+    line-height: 24px;
```

Comment:
If possible, Use rems here to maintain consistency

Reply:
Yups. Changed 👍